### PR TITLE
Explicitly remove timeouts from Http client

### PR DESCRIPTION
**Pull request by KhalilBell**

### CODE REVIEW

**Review by eBardX — COMMENTED** [review]
Setting the timeout on the client did not produce the desired effect?

**Comment by KhalilBell** [comment]
> Setting the timeout on the client did not produce the desired effect?

After searching that was the advised way of removing timeouts on client (also I didn't find the same granularity on timeouts on the client)

File: upload.go [hunk 1]
```diff
@@ -302,7 +302,13 @@ func (ua *uploadAction) uploadBuild(retryAllowed bool) (bool, error) {
...
 	defer file.Close()
 
-	client := &http.Client{}
+	client := &http.Client{
+		Transport: &http.Transport{
+              		TLSHandshakeTimeout: 0,
+              		ResponseHeaderTimeout: 0,
+              		ExpectContinueTimeout: 0,
+            	},
+	}
```
Comment (eBardX) [inline upload.go:311]: nit: I’m not a big fan of extraneous commas in arrays and structs (like you have on lines 309 and 310).

**Review by eBardX — CHANGES_REQUESTED** [review]
Please update the agent version in `main.go` to 2.1.1 as well as `CHANGELOG.md`. (Bonus points if you can modify the change log to my satisfaction. 😉)

**Comment by eBardX** [comment]
So close on the change log! I added a missing link near the end so that the correct tags can be compared.

**Review by eBardX — APPROVED** [review]

**Pull request merged**